src/applications/static-pages/view-modify-dependent/686-cta/containers/Form686CTA.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import { connect } from 'react-redux';
+
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+const Form686CTA = props => {
+  let content;
+  if (props.includedInFlipper === undefined) {
+    content = <LoadingIndicator message="Loading..." />;
+  } else if (props.includedInFlipper === true) {
+    content = (
+      <>
+        <p className="vads-u-font-size--xl vads-u-font-family--serif vads-u-font-weight--bold">
+          How do I add or change a dependent on my VA disability benenfits?
+        </p>
+        <p>
+          You could receive additional disability compensation for a spouse,
+          child or parent, if you are eligible for VA disability compensation,
+          and you have a combined disability rating of at least 30%. Adding a
+          dependent may make you eligible to receive a higher compensation
+          payment (also called a benefit rate).
+        </p>
+        <a className="vads-u-display-block">
+          Learn about adding dependents to your VA disability benefits
+        </a>
+        <p className="vads-u-margin-top--1p5">
+          To file a claim for additional disability compensation for a child or
+          spouse, you can submit an Application Request To Add And/Or Remove
+          Dependents (VA Form 21-686c) at the link below. Note that you may need
+          to provide more information or forms along with your claim.
+        </p>
+        <a
+          href="/view-change-dependents/add-remove-form-686c"
+          className="usa-button"
+        >
+          Add or change dependents on your VA Benefits
+        </a>
+      </>
+    );
+  } else {
+    content = <></>;
+  }
+  return <div>{content}</div>;
+};
+
+const mapStateToProps = store => ({
+  user: store.user,
+  includedInFlipper: toggleValues(store)[
+    FEATURE_FLAG_NAMES.vaViewDependentsAccess
+  ],
```

Comment:
The name is your call but I think the usually convention is to try to portray how the feature is represented in the UI - `show686Link` or something like that. Here's an example of a dynamic link in `/find-locations` that was recently implemented - https://github.com/department-of-veterans-affairs/vets-website/blob/c29d389ed142049e7fe32c38b459d48929394f56/src/applications/facility-locator/containers/VAMap.jsx#L779.

Reply:
Love this, much more clear. Going to update this to `showContent`